doc/source/overview/config_files.md

```diff
@@ -634,18 +375,21 @@ layers = [-1, -2, -3, -4, -5, -6, -7, -8]
 
 % TODO: Add a second example demo-ing loading of state (once implemented). e.g. for online or learned kernel.
 
-## Advanced usage
+%## Advanced usage
 
-### Validating config files
+(validation)=
+## Validating config files
```

Comment:
This means the whole section is commented out?

Reply:
I've just removed the "Advanced usage" header, and promoted "Validating config files", since we didn't have any other level 3 headings under "Advanced usage". Can change in future if we add more "Advanced topics".